src/tools/pub/cmd/pub-token.md

```diff
@@ -0,0 +1,82 @@
+---
+title: dart pub token
+description: Manage authentication tokens for package repositories.
+---
+
+_pub_ _token_ is one of the subcommands of the [pub command](/tools/pub/cmd).
+
+It is used to update a store of credential tokens used for authenticating
+against third-party servers when [publishing](pub-lish) a package and when
+[retrieving](pub-get) dependencies.
+
+The token store is stored in a
+[user-wide config dir](https://github.com/dart-lang/cli_util/blob/71ba36e2554f7b7717f3f12b5ddd33751a4e3ddd/lib/cli_util.dart#L88-L118). 
+
+It has three subcommands: `add`, `list` and `remove`.
+
+If you try to `dart pub get` and have a [dependency](/tools/pub/dependencies) hosted
+on a private repository you will be asked to provide credentials:
+
+```terminal
+$ dart pub get
+Resolving dependencies... 
+https://some-package-repo.com/my-org/my-repo package repository requested authentication! You can provide credential using:
+    pub token add https://some-package-repo.com/my-org/my-repo
+
+Go to https://some-package-repo.com and log in to obtain your token. 
+```
+
+The last line is a message the server can provide to help you obtaining a token.
+Some servers might not provide such a message.
+
+## Adding credentials `dart pub token add`
+
+To enter the credentials use the `add` subcommand, 
```

Comment:
I wonder if we should spell out the entire command, like `dart pub token add`

Reply:
Done